erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json

```diff
@@ -423,6 +423,7 @@
   {
    "fieldname": "rejected_warehouse",
    "fieldtype": "Link",
+   "ignore_user_permissions": 1,
```

Comment:
Why do we need to ignore user permissions here?

Reply:
- Create the User Permission for the Warehouse and enable the IS Default
- Then try to create the Purchase Receipt
- System auto set the same value in the Accepted warehouse and rejected warehouse based on the User Permission
- On save, you will get the error that "Accepted warehouse and Rejected warehouse cannot be same", even though you have not set the rejected warehouse.

Reply:
Okay, so in oder to be able set a _different_ rejected warehouse, we need to ignore the user permission. Makes sense.